wippersnapper-config-builder.js

```diff
@@ -1536,6 +1536,11 @@ function saveModalData() {
         componentConfig.i2cDeviceName = componentId;
         componentConfig.i2cDeviceAddress = i2cAddress;
 
+        // Add deviceId field if it exists in the component template
```

Comment:
doesn't do what the comment says

Reply:
It doesn’t add the deviceId field, it adds the isGps  field